dispatch/static/manager/src/js/api/dispatch.js

```diff
@@ -92,6 +105,9 @@ var DispatchAPI = {
       }
 
       return postRequest('auth/token', null, payload)
+    },
+    deleteToken: (token) => {
+      return delRequest('auth/token', null, token, token)
```

Comment:
If we make the backend change above, we don't need to include the token in the payload:

```python
return delRequest('auth/token', null, null, token)
```